src/vs/workbench/services/update/browser/updateService.ts

```diff
@@ -97,6 +97,12 @@ export class BrowserUpdateService extends Disposable implements IUpdateService {
 	async _applySpecificUpdate(packagePath: string): Promise<void> {
 		// noop
 	}
+
+	// --- Start Positron ---
+	updateActiveLanguages(languages: string[]): void {
```

Comment:
I did a quick test locally and this seems like it could be used for Workbench as well. It would adding in a scheduled timer for this service as it only runs the update check once at startup.